celery/contrib/django/task.py

```diff
@@ -0,0 +1,24 @@
+import functools
+
+try:
+    from django.db import transaction
+except ImportError:
+    transaction = None
```

Comment:
Added this try/catch to let us generate the API reference page. 

I'm not a big fan of this as it might hide some misconfiguration in the app, the error will happen when the method is called rather than at start-up. 

Is there any pre-existing way of dealing with 3rd party libs that are not present when generating the documentation?

Reply:
Amazing, thanks for the pointer!